### ISSUE TITLE
Bulleted list - Bullet point a11y graphic improvements

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#3607]
**Describe the bug**
The bulleted list 'bullet' point has a couple of accessibility bugs. This uses a background image for the 'bullet'. This doesn't change colour when the user changes their colour preferences in FireFox and therefore when the user selects a black/dark background the bullet cannot be seen. The same problem occurs when using a high contrast mode on PC. 

https://bbc.github.io/psammead/iframe.html?id=components-bulletedlist--default

The can be addressed by creating a bullet out of CSS rather than using a background image, such as... (note we don't want to use CSS list style as this degrades the screen reader ux with output such as 'bullet' to the user)

border: .1875rem solid #3f3f42;
border-radius: 50%;
content: '';
left: -1rem;
position: absolute;
top: .5rem;

See implementation here https://bbc.github.io/accessibility-news-and-you/guides/screen-reader-ux.html

**Expected behaviour**
I should be able to see the bullet point when I change my colour preferences to a black background in FireFox and also High contrast mode on PC with a black background.

**Testing notes**
2 person dev - all1y swarm required.

**Comment by karinathomasbbc** [comment]
Another option is to manipulate the svg in high contrast mode - will explore the options once I start working on this properly

**Comment by greenc05** [comment]
Good thinking @karinathomasbbc however that won't work for all colours when the user changes colour preferences in Firefox, the icon will remain black and if a black background is used it will be 'invisible'

**Comment by aeroplanejane** [comment]
What have they done with unordered lists on webcore and can we do same / similar? Just leaving this here... https://www.bbc.co.uk/news/explainers-53431026

**Comment by greenc05** [comment]
Good find @aeroplanejane however that's exactly what we didn't want e.g. they use a CSS list type bullet which means that 'bullet' is read out to screen readers in addition to everything else.

**Comment by greenc05** [comment]
The values on the issue where appropraite for this site https://bbc.github.io/accessibility-news-and-you/guides/screen-reader-ux.html Please check that the rem values and colours are correct @pharingee 

In the dev a11y swarm please test with:
- all our supported screen readers and that everything is as expected
- I should be able to see the bullet point when I change my colour preferences to a black background in FireFox 
- I should be able to see the bullet point when I view in High contrast mode on PC with a black background
- Should look ok with no CSS e.g. I can see a 'bullet' for each list item
- View the feature with text resized to 200% - the 'bullet' should be proportionally larger 
- View the feature with the page zoomed to 200%- the 'bullet' should be zoomed proportionally
